### PR TITLE
Ci/deploy on pull 

### PR DESCRIPTION
**Pull request by sickinga**
This commit enables firebase to deploy the website after a pull request

### CODE REVIEW

File: .github/workflows/firebase-hosting-merge.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm run build
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
```
Comment (LeaFranz) [inline .github/workflows/firebase-hosting-merge.yml:17]: Does this add the api keys?

Comment (sickinga) [reply .github/workflows/firebase-hosting-merge.yml:17]: no this is basically only a token for the github permissions

Comment (LeaFranz) [reply .github/workflows/firebase-hosting-merge.yml:17]: Where do we add it then? Loading the weather doesn't work yet on the deployed application

Comment (sickinga) [reply .github/workflows/firebase-hosting-merge.yml:17]: They were added

**Review by LeaFranz — APPROVED** [review]

**Pull request merged**